### PR TITLE
renaming mss to msui, refactored

### LINKED ISSUE #1197: python-mss overwriting mss shortcut
If MSS is installed and the user types "mss" in their console, MSS starts.
If then python-mss is installed afterwards and the user types "mss" into their console, python-mss starts and takes a screenshot.

Not particularly troublesome but I thought I would document this behaviour.

### LINKED ISSUE #1430: refactoring for consistant nameings
We have currently a mss command which is the name of the whole Software System. This starts the graphical interface.

We want in the future a mss command which is a CLI Interface for all parts of MSS. This has then sections for all components.

While we already have mscolab, mswms, we now need a msui. The naming convention is already used in our release notes and have to become redrawn to its source.

Therefore we set in 6.1.0 a deprecation warning and do the refactoring for 7.0.0

### PR DESCRIPTION
**Pull request by ReimarBauer**

### CODE REVIEW

File: docs/samples/automation/retriever.py [hunk 1]
```diff
@@ -43,7 +43,7 @@
 from mslib.utils.units import units
 import mslib.msui
 import mslib.msui.mpl_map
-import mslib.msui.mss_qt
+import mslib.msui.msui_qt
```
Comment (joernu76) [inline docs/samples/automation/retriever.py]: -> qt ?

File: docs/samples/config/wms/mss_wms_settings.py.sample [hunk 2]
```diff
@@ -151,7 +151,7 @@ register_horizontal_layers = [
 
     # MSS-Chem chemistry forecasts
     #
-    # The MSS-Chem project (http://mss-chem.rtfd.io) provides an easy way to
+    # The MSS-Chem project (http://msui-chem.rtfd.io) provides an easy way to
```
Comment (joernu76) [inline docs/samples/config/wms/mss_wms_settings.py.sample]: No!

File: docs/samples/plugins/navaid.rst [hunk 3]
```diff
@@ -27,7 +27,7 @@ waypoint is more than 500 nm.
 Installation
 ~~~~~~~~~~~~
 
-1. Copy :download:`navaid.py <samples/plugins/navaid.py>` to a PYTHONPATH directory e.g. .config/mss
+1. Copy :download:`navaid.py </samples/plugins/navaid.py>` to a PYTHONPATH directory e.g. .config/mss
```
Comment (joernu76) [inline docs/samples/plugins/navaid.rst:30]: intended?

File: docs/usage.rst [hunk 4]
```diff
@@ -1,13 +1,13 @@
-mss - User Interface
-======================================
+msui   (MSS - User Interface)
```
Comment (joernu76) [inline docs/usage.rst]: Mission Support User Interface ?

**Review by joernu76 — CHANGES_REQUESTED** [review]

File: docs/samples/config/wms/mss_wms_settings.py.sample (hunk 2 above)
Comment (ReimarBauer) [reply docs/samples/config/wms/mss_wms_settings.py.sample]: ohhhh

File: docs/samples/plugins/navaid.rst (hunk 3 above)
Comment (ReimarBauer) [reply docs/samples/plugins/navaid.rst:30]: yes, this was broken, it didn't find the file. without the right location it tries ./samples/plugins/samples/plugins/navaid.py
That's the reason why we have only a download sign on https://mss.readthedocs.io/en/stable/samples/plugins/navaid.html but not a file connected

File: docs/samples/automation/retriever.py (hunk 1 above)
Comment (ReimarBauer) [reply docs/samples/automation/retriever.py]: I moved this to mslib.utils.qt

**Comment by ReimarBauer** [comment]
This may not a new problem. I got it on storing a new msui_settings.json
```
Traceback (most recent call last):
  File "/home/reimar/MASTER/MSS/mslib/msui/msui.py", line 881, in restart_application
    self.add_plugins()
  File "/home/reimar/MASTER/MSS/mslib/msui/msui.py", line 487, in add_plugins
    self.add_plugin_submenu("CSV", "csv", load_from_csv, picker_default, plugin_type="Import")
  File "/home/reimar/MASTER/MSS/mslib/msui/msui.py", line 505, in add_plugin_submenu
    raise ValueError(f"'{action_name}' has already been set!")
ValueError: 'actionImportFlightTrackCSV' has already been set!
```

**Comment by ReimarBauer** [comment]
**REMARK** 
We have also to update the docker scripts in the other repositories, after release.

File: localbuild/bld.bat [hunk 5]
```diff
@@ -1,7 +1,7 @@
 set MENU_DIR=%PREFIX%\Menu
 if not exist %MENU_DIR% mkdir %MENU_DIR%
 if errorlevel 1 exit 1
-copy %SRC_DIR%\Menu\mss_shortcut.json %MENU_DIR%\mss_shortcut.json
+copy %SRC_DIR%\Menu\mss_shortcut.json %MENU_DIR%\msui_shortcut.json
```
Comment (joernu76) [inline localbuild/bld.bat]: The original shortcut should be renamed as well?

File: localbuild/bld.bat [hunk 6]
```diff
@@ -1,7 +1,7 @@
...
 if errorlevel 1 exit 1
 copy %SRC_DIR%\Menu\mss.ico %MENU_DIR%\mss.ico
```
Comment (joernu76) [inline localbuild/bld.bat:6]: mss or msui?

File: mslib/msui/qt5/ui_mainwindow.py [hunk 7]
```diff
@@ -12,7 +12,7 @@
 
 class Ui_MSSMainWindow(object):
```
Comment (joernu76) [inline mslib/msui/qt5/ui_mainwindow.py]: Ui_MSUIMainWindow?

Comment (joernu76) [reply mslib/msui/qt5/ui_mainwindow.py]: Better to fix ui_mainwindow.ui and regenerate.

File: localbuild/bld.bat (hunk 6 above)
Comment (joernu76) [reply localbuild/bld.bat:6]: then the msui_shortcut.json needs to be adapted as well.

**Comment by joernu76** [comment]
Should the docs/samples/config/wms/mss_wms* stuff be renamed consistently to docs/samples/config/mswms/mswms_* ?

File: localbuild/bld.bat (hunk 5 above)
Comment (ReimarBauer) [reply localbuild/bld.bat]: yes, the "mss" shortcut is currently deprecated.

File: localbuild/bld.bat (hunk 6 above)
Comment (ReimarBauer) [reply localbuild/bld.bat:6]: ok, will have a look

**Comment by ReimarBauer** [comment]
> Should the docs/samples/config/wms/mss_wms* stuff be renamed consistently to docs/samples/config/mswms/mswms_* ?

Yes, the dir wms needs to be renamed. I'm not sure what we should do with the different settings files. 
maybe we should have a sub dir for the idea they solve and have them always named mswms_settings.py.sample and a two liner to describe what that is.

**Review by joernu76 — APPROVED** [review]

**Pull request merged**